src/drivers/code_hosting/bitbucket.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+
+# Returns the source for a new Bitbucket pull request
+function bitbucket_source {
+  local repository=$1
+  local branch=$2
+
+  local sha=$(git log --format="%H" -1 | cut -c-12)
+  echo "$repository:$sha:$branch" | sed 's/\//%2F/g' | sed 's/\:/%3A/g'
+}
+
+
+function create_pull_request {
+  local repository=$1
+  local branch=$2
+
+  open_browser "https://bitbucket.org/$repository/pull-request/new?source=$(bitbucket_source "$repository" "$branch")"
+}
+
+
+function show_repo {
+  local repository=$1
+
+  open_browser "https://bitbucket.org/$repository"
```

Comment:
Yes, but I also like the readability of the full URL here. 
```
open_browser "$repository_base_url/pull-request/new?source=$(bitbucket_source "$repository" "$branch")"
``` 
doesn't make it clear to me what exact URL is being opened.

```
open_browser "https://bitbucket.org/$repository/pull-request/new?source=$(bitbucket_source "$repository" "$branch")"
```
seems more readable.

Plus, this would result in more code, and more lines of code. All of this together makes me  hesitant to implement this suggestion.